appinfo/register_command.php

```diff
@@ -1,11 +1,12 @@
 <?php
-/**
- * @var Symfony\Component\Console\Application $application
- */
 
 // @codeCoverageIgnoreStart
 $config = \OC::$server->getConfig();
 $ocVersion = $config->getSystemValue('version', '');
+
+/**
+ * @var Symfony\Component\Console\Application $application
+ */
```

Comment:
Moved this to above the var that it describes. That at least stops phpstan complaining about the PHPdoc in the wrong place.